app/views/billing_subscriptions/_new_modal.html.erb

```diff
@@ -1,35 +1,37 @@
 <%- # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later -%>
 <div class='modal skinny' id='newBillingSubscriptionModal'>
-	<%= render 'common/modal_header', title: "Movin' on up!" %>
```

Comment:
Could you undo all of the formatting changes here? We don't have formal formatting rules on this repo and it makes it unclear what actually changed.

Reply:
Okay!